components/monitoring/base/prometheus-servicemonitors.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  namespace: appstudio-workload-monitoring
+  name: prometheus
+  labels:
+    prometheus: appstudio-kcp-workload
+spec:
+  endpoints:
+  - bearerTokenSecret:
+      key: ""
```

Comment:
why is it empty?

Reply:
this is local to prometheus installation, we are accessing this through service and cert file directly.

Reply:
I thought oauth2-proxy needed a token in the requests? 🤔

Reply:
:thinking: this works in local, and the pre-kcp stage. I will re-verify this.